packages/scripts/src/tasks/providers/android.ts

```diff
@@ -1,11 +1,92 @@
+import * as path from "path";
+import { Options, run } from "cordova-res";
+import fs from "fs";
+import { ROOT_DIR } from "../../paths";
+import { logError } from "../../utils";
+
+const set_splash_image = async (assetsFolderPath: string, subPath: string) => {
+  const androidAssetsPath = path.join(path.resolve(assetsFolderPath), subPath);
```

Comment:
This may need a tweak if you go ahead and allow users to specify icon and splash asset paths explicitly in config. The config generator automatically hardcodes relative paths, however I expect it will miss the app-data/assets part of the path so that might need fixing